library/output-cjs/package.json

```diff
@@ -1 +0,0 @@
-{ "type": "commonjs" }
```

Comment:
For now, recommend just keeping this as-is so we continue to build CJS versions that work correctly.  I agree we should drop support soon, but in a different PR.